src/platform/plugins/shared/dashboard/public/plugin.tsx

```diff
@@ -327,7 +337,7 @@ export class DashboardPlugin
     setKibanaServices(core, plugins);
 
     untilPluginStartServicesReady().then(() => {
-      buildAllDashboardActions({
+      registerActions({
```

Comment:
Do we still need this nested under an `untilPluginStartServicesReady`? On the line above we synchronously set the plugin start services, so as a rule this will just add a layer of `promise.resolve`.